arkane/statmech.py

```diff
@@ -510,6 +510,7 @@ def load(self, pdep=False, plot=False):
             logging.debug('         E0 (0 K) = {0:g} kcal/mol'.format(e0 / 4184.))
 
         conformer.E0 = (e0 * 0.001, "kJ/mol")
+        self.species.props['atom_corrections_298K'] = atom_corrections_298K
```

Comment:
from the  tests:
`AttributeError: 'rmgpy.species.TransitionState' object has no attribute 'props'`